docs/_deployments/serverless-framework.md

```diff
@@ -0,0 +1,346 @@
+
+**1. Install the Serverless Framework CLI**
+
+The Serverless tools are available as a Command Line Interface (CLI) and can be [installed on macOS, Windows, or Linux](https://www.serverless.com/framework/docs/getting-started/). You can install it using npm with the command:
+
+```shell
+npm install --save-dev serverless
+```
+
+> 💡 Alternatively, you can [globally install the Serverless CLI](https://www.serverless.com/framework/docs/getting-started/) using the command `npm install -g serverless`.
+
+Once the installation is complete, you can test the Serverless CLI by displaying the commands available to you:
+
+```shell
+npx serverless help
+```
+
+You're now set up with the Serverless tools! Let's move on to preparing your Bolt app to run as an AWS Lambda function.
+
+---
+
+### Get a Bolt Slack app
+
+If you haven't already built your own Bolt app, you can use our [Getting Started guide][getting-started-guide] or clone the template app below:
+
+```shell
+git clone https://github.com/slackapi/bolt-js-getting-started-app.git
+```
+
+After you have a Bolt app, navigate to its directory:
+
+```shell
+cd bolt-js-getting-started-app/
+```
+
+Now that you have an app, let's prepare it for AWS Lambda and the Serverless Framework.
+
+---
+
+### Prepare the app
+
+**1. Prepare the app for AWS Lambda**
+
+In this section, we'll update your Bolt app to respond to a Lambda function instead of listening for HTTP requests. We'll do this by creating a custom Bolt receiver for Lambda.
+
+First, install the official [AWS Serverless Express](https://github.com/awslabs/aws-serverless-express) module to transform Express HTTP requests to Lambda function events:
+
+```bash
+npm install aws-serverless-express
+```
+
+Next, update the [source code that imports your modules](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/app.js#L1) in `app.js` to require Bolt's Express receiver and the AWS Serverless Express module:
+
+```javascript
+const { App, ExpressReceiver } = require('@slack/bolt');
+const awsServerlessExpress = require('aws-serverless-express');
+```
+
+Then update the [source code that initializes your Bolt app](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/app.js#L3-L7) to create a custom receiver using AWS Serverless Express:
+
+```javascript
+// Initialize your custom receiver
+const expressReceiver = new ExpressReceiver({
+  signingSecret: process.env.SLACK_SIGNING_SECRET,
+  // The `processBeforeResponse` option is required for all FaaS environments.
+  // It allows Bolt methods (e.g. `app.message`) to handle a Slack request
+  // before the Bolt framework responds to the request (e.g. `ack()`). This is
+  // important because FaaS immediately terminate handlers after the response.
+  processBeforeResponse: true
+});
+
+// Initializes your app with your bot token and signing secret
```

Comment:
Actually, we don't directly pass signing secret here. What do you think about changing like this (in a better language)?
```suggestion
// Initializes your app with your bot token and the AWS Lambda ready receiver
```